client2/src/game/Game.js

```diff
@@ -148,6 +150,26 @@ class Game extends Phaser.Scene {
          * @type {Object}
          */
         this.lightSources = {};
+
+        /**
+         * ChatInput status, determines whether chatInput is closed or opened.
+         * @type {Boolean}
+         */
+        this.chatInputStatus = false;
+
+        /**
+         * A list of PubSub subscription
+         * @TODO Move to a separate file
+         * @type {Array.<PubSub>}
+         */
+        this.subs = [
```

Comment:
I think we should move this to a separate file to avoid cluttering, as this could be a huge list in the future.

Reply:
Yeah good idea